lib/twilio/preflight/preflight.ts

```diff
@@ -292,7 +292,7 @@ export class PreflightTest extends EventEmitter {
     const report: PreflightTest.Report = {
       callSid: this._callSid,
       edge: this._edge,
-      iceCandidateStats: this._rtcIceCandidateStatsReport.iceCandidateStats,
+      iceCandidateStats: this._rtcIceCandidateStatsReport?.iceCandidateStats ?? [],
```

Comment:
Nitpick, would prefer to pull this into it's own variable.

```
const iceCandidateStats = this._rtcIceCandidateStatsReport?.iceCandidateStats ?? [];

const report = {
  ...,
  iceCandidateStats,
  ...,
};
```

Reply:
I don't see a lot of benefits for doing that now. Maybe if we have to use the variable multiple times we can consider doing it.